util/design/lib/OtpMemImg.py

```diff
@@ -210,37 +203,33 @@ def merge_part_data(self, part):
                 log.warning("Partition does not contain any items.")
 
         # Check if partition name exists in memory map and get index
```

Comment:
Super-nit: This is no longer getting an index. (I only noticed because I searched for the comment to find the right place in the file!)

Reply:
ah yeah right - removed that part of the sentence.